src/main/kotlin/com/terraformation/backend/nursery/db/Exceptions.kt

```diff
@@ -12,3 +14,11 @@ class BatchStaleException(val batchId: BatchId, val requestedVersion: Int) :
 
 class BatchInventoryInsufficientException(val batchId: BatchId) :
     IllegalArgumentException("Withdrawal quantity can't be more than remaining quantity")
+
+class CrossOrganizationNurseryTransferNotAllowedException(
+    val facilityId: FacilityId,
+    val destinationFacilityId: FacilityId
+) :
+    MismatchedStateException(
+        "Cannot transfer from $facilityId to $destinationFacilityId because they are in " +
```

Comment:
just a thought - should we handle this as destinationFacilityId not found ?

Reply:
I thought about that, but it seems like the wrong error if you're in both facilities' organizations and can see that both facilities exist.

Reply:
Got it, I also missed the requirePermissions check on the destination facilityId.. that should handle the non-membership case. Thanks.